src/main/java/bio/terra/service/filedata/google/gcs/GcsPdao.java

```diff
@@ -336,4 +325,24 @@ private void fileAclOp(AclOp op, Dataset dataset, List<String> fileIds, Map<IamR
             return file;
         };
     }
+
+    /**
+     * Extract the bucket from a gs path.
+     */
+    public static String extractBucketFromPath(final String path) {
+        if (!path.startsWith("gs://") && StringUtils.countMatches(path, ',') < 3) {
```

Comment:
Why is this code checking for `,`? Did you mean `/`? I would've expected something like
```
        if (!path.startsWith("gs://") || !StringUtils.countMatches(path, '/') < 3) {
```

Although a path with just a bucket is still a valid gs path, and doesn't need a trailing `/`.

Reply:
Ugh...shoot, I don't know what happened here...good catch.  This is telling me I need a unit test here.  Will do